share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -189,7 +189,7 @@ spack:
   - cabana +rocm
   - caliper +rocm
   - chai ~benchmarks +rocm
-  - ecp-data-vis-sdk +paraview +vtkm +rocm
+  - ecp-data-vis-sdk +ascent +adios2 +hdf5 +paraview +pnetcdf +sz +vtkm +zfp +rocm
```

Comment:
Only ascent, paraview, and vtk-m have ROCm support, the other packages are integrations that are useful to have in conjunction with the ROCm enabled packages.